### ISSUE TITLE
You do not show all the codes!

### ISSUE THREAD

**Issue by jingchangshi** [ChongPengGeotech/LOQUAT#1]
You do not show all the codes!

For example, where is the implementation of `SortArrayFloat4`? It is declared in `loquat_grid.h` but no implementation is found!

Really bad "OPEN SOURCE" codes!

**Comment by dineshadepu** [comment]
Yeah. Peng, it would really be helpful, to put some more code here. My sincere request. 

May be you can exclude the nnps, but basic particle interaction is also not provided. 

Please kindly provide.

**Comment by yuantangjin** [comment]
> Yeah. Peng, it would really be helpful, to put some more code here. My sincere request.
> 
> May be you can exclude the nnps, but basic particle interaction is also not provided.
> 
> Please kindly provide.



> You do not show all the codes!
> 
> For example, where is the implementation of `SortArrayFloat4`? It is declared in `loquat_grid.h` but no implementation is found!
> 
> Really bad "OPEN SOURCE" codes!

I think these file has been provided in another way. I don't know if I understand it correctly but I have found the file "libparticle_interaction.a" and the file "libloquat_grid.a", Does this mean that we can only make calls in the form of library files, and cannot modify or view the source code?

**Comment by ChongPengGeotech** [comment]
Recently, we were reminded by an anonymous reviewer that there are some issues raised. Sorry, we did not maintain the repository for a long time. In the new update, all the source files were updated and the Boost dependency was removed.
